instrgen/driver/instrgen_test.go

```diff
@@ -61,8 +96,8 @@ func Test(t *testing.T) {
 					f2, err2 := os.ReadFile(expectedFile)
 					require.NoError(t, err2)
 					if !assert.True(t, bytes.Equal(f1, f2)) {
-						fmt.Println(k)
-						failures = append(failures, k)
+						fmt.Println(file)
```

Comment:
The print statement can be merged into the assert command.

```suggestion
					if !assert.True(t, bytes.Equal(f1, f2), file) {
						failures = append(failures, k)
```

Reply:
ok, thx